tests/testthat/test-fixed_regex_linter.R

```diff
@@ -32,7 +32,7 @@ test_that("fixed_regex_linter skips allowed usages", {
 
 test_that("fixed_regex_linter blocks simple disallowed usages", {
   linter <- fixed_regex_linter()
-  msg <- rex::rex("For static regular expression patterns, set `fixed = TRUE`.")
+  msg <- rex::rex("This regular expression is static")
```

Comment:
let's include the static equivalent part of the regex in the test message, that's the most important part IMO

Reply:
I've added a patrick test for all patterns used throughout the rest of the test suite, to keep diff sizes low in case we change lint messages down the line.